README.md

```diff
@@ -0,0 +1,344 @@
+
+## Creating a new Blazor page
+
+If you have never used Blazor before, just hit the `Run` button and have a look at the template that has been generated.
+It's a great start and we will be re-using same parts of it later on.
+
+Let's add our own page first though. In your Solution Explorer you'll see a `Pages` folder.
+Right-click it and add a `Razor Component`. Those are files that combine the HTML of your page with C# code.
+
+![05_add_a_new_component.jpg](https://mongodb-devhub-cms.s3.us-west-1.amazonaws.com/05_add_a_new_component_6476440c47.jpg)
+![06_name_the_new_component.jpg](https://mongodb-devhub-cms.s3.us-west-1.amazonaws.com/06_name_the_new_component_8ecc33dea7.jpg)
+
+Now, replace the content of the file with the following code. Explanations can be read inline in the code comments.
+
+```csharp
+// The `page` attribute defines how this page can be opened.
+@page "/news"
+
+// The `MongoDB` driver will be used to connect to your Atlas cluster.
+@using MongoDB.Driver
+// `BSON` is a file format similar to JSON. MongoDB Atlas documents are BSON documents.
+@using MongoDB.Bson
+// You need to add the `Data` folder as well. This is where the `News` class resides.
+@using CryptoNewsApp.Data
+
+// The page title is what your browser tab will be called. 
+<PageTitle>News</PageTitle>
+
+// Let's add a header to the page.
+<h1>News</h1>
+
+// And then some data.
+// This is just a simple table contains news and their date.
+@if (_news != null)
+{
+    <table class="table">
+        <thead>
+        <tr>
+            <th>News</th>
+            <th>Date</th>
+        </tr>
+        </thead>
+        <tbody>
+        // Blazor takes this data from the `_news` field that we will fill later on.
+        @foreach (var newsEntry in _news)
+        {
+            <tr>
+                <td>@newsEntry.Title</td>
+                <td>@newsEntry.Date</td>
+            </tr>
+        }
+        </tbody>
+    </table>
+}
+
+// This part defines the code that will be run when the page is loaded. It's basically 
+// what would usually be PHP in a non-Blazor environment.
+@code {
+    
+    // The `_news` field will hold all our news. We will have a look at the `News`
+    // class in just a moment.
+    private List<News>? _news;
+
+    // `OnInitializedAsync()` gets called when the website is loaded. Our data
+    // retrieval logic has to be placed here.
+    protected override async Task OnInitializedAsync()
+    {
+        // First, we need to create a `MongoClient` which is what we use to
+        // connect to our cluster.
+        // The only argument we need to pass on is the connection string you
+        // retrieved from Atlas. Make sure to substitute the password with
+        // the actual password.
+        var mongoClient = new MongoClient("mongodb+srv://dbuser:dbUserPassword@cluster0.pl7egwa.mongodb.net/?retryWrites=true&w=majority");
```

Comment:
I don't know if this comes later, but can we use environment variables instead of hard-coding the connection string into the application?